File: angularclient/src/app/model/Settings/ContactInfo.ts
export class ContactInfo {
  id;
  function;
  name;
  phone: number;
  mail;


  constructor(id = '', Function = '', Name = '', Phone: number = 0, Mail = '') {
    this.id = id;
    this.function = Function;
    this.name = Name;
    this.phone = Phone;
    this.mail = Mail;
  }
}

File: ra-back/src/main/java/pl/raptors/ra_back/domain/settings/ContactInfo.java
package pl.raptors.ra_back.domain.settings;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@NoArgsConstructor
@AllArgsConstructor
@Data
@Document(collection = "contactInfo")
public class ContactInfo {
    @Id
    private String _id;
    private String Function;
    private String Name;
    private Long Phone;
    private String Mail;

}
